src/bylines/index.js

```diff
@@ -35,9 +35,19 @@ const BylinesSettingsPanel = () => {
 	}
 	// Byline change handler.
 	const handleBylineChange = value => {
-		const tags = value.match( /<[^>]+>/g );
-		if ( tags && tags.some( tag => ! tag.startsWith( '<Author' ) && ! tag.startsWith( '</Author' ) ) ) {
-			alert( __( 'Only the <Author> tag is allowed.', 'newspack-plugin' ) ); // eslint-disable-line no-alert
+		const tags = value.match( /\[[^\]]+\]/g );
```

Comment:
Non-blocking: Looking at this again after a while, this could use a code comment to clarify what we are searching for.